README.md

```diff
@@ -42,7 +44,325 @@
+│   │   │       ├── nginx-config
+│   │   │       ├── nginx-config.templ
+│   │   │       ├── nginx-filter.conf
+│   │   │       ├── postgres-filter.conf
+│   │   │       ├── postgresql-filter-extra.conf
+│   │   │       ├── provisioner.sh
+│   │   │       └── syslog-filter.conf
+│   │   ├── README.md
+│   │   └── script.sh
+│   ├── nat-image
+│   │   ├── nat-config.sh
+│   │   ├── nat-template.json
+│   │   └── nat-template.json.templ
+│   ├── shared
+│   │   ├── filebeat-config.yml
+│   │   ├── filebeat-config.yml.templ
+│   │   ├── google-creds-sandbox.json
+│   │   ├── google-creds-staging.json
+│   │   ├── google-creds.json
+│   │   ├── metricbeat-config.yml
+│   │   ├── metricbeat-config.yml.templ
+│   │   └── slack.sh
+│   └── vault-image
+│       ├── README.md
+│       ├── mrm-vault-config-file.hcl
+│       ├── mrm-vault-policy.hcl
+│       ├── renew-token
+│       ├── startup-script.sh
+│       ├── startup-script.sh.templ
+│       ├── vault-provisioner.sh
+│       ├── vault-template.json
+│       ├── vault-template.json.templ
+│       └── vault.service
+├── production_env.sh
+├── pull_request_template.md
+├── sandbox_env.sh
+├── script_linux.sh
+├── script_mac.sh
+├── secrets
+│   ├── barman_rsa.pub
+│   ├── github_key_private
+│   ├── github_key_private.pub
+│   ├── logstash-forwarder.crt
+│   ├── logstash-forwarder.key
+│   ├── postgres_rsa
+│   ├── postgres_rsa.pub
+│   ├── star_andela_cert.pem
+│   └── star_andela_key.pem
+├── slack.sh
+└── terraform
+    ├── README.md
+    ├── backend-instance-group.tf
+    ├── database.tf
+    ├── firewall-config.tf
+    ├── frontend-instance-group.tf
+    ├── global-instance-configs.tf
+    ├── google-creds-sandbox.json
+    ├── google-creds-staging.json
+    ├── instance-group-template.tf
+    ├── ip-addresses.tf
+    ├── providers.tf
+    ├── routes.tf
+    ├── terraform-init
+    ├── terraform-init.templ
+    ├── variables.tf
+    ├── variables.tf.templ
+    └── vpc-config.tf
+
+```
+
+### Creating Images
+The infrastructure requires custom images to make the deployment of the application faster. The custom images created include, 
+
+- [backend-image](https://github.com/andela/mrm-deployment-scripts/tree/develop/images/backend-image)
+- [frontend-image](https://github.com/andela/mrm-deployment-scripts/tree/develop/images/frontend-image)
+- [monitoring](https://github.com/andela/mrm-deployment-scripts/tree/develop/images/monitoring)
+- [nat-image](https://github.com/andela/mrm-deployment-scripts/tree/develop/images/nat-image)
+- [vault-image](https://github.com/andela/mrm-deployment-scripts/tree/develop/images/vault-image)
+
+#### Creating image files
+
+The examples below use the script_linux.sh however, the same applies to script_mac.sh. The following commands will create the appropriate files from template files to execute the packer.
+
+###### Creating files for all images
+The command below will create all files for the images
+
+`source script_linux.sh sandbox create image all`
```

Comment:
How does a new user get the `script_linux`.sh file and keep it updated @philophilo?